src/styles/bootstrap-theme/colors.module.scss

```diff
@@ -27,18 +27,31 @@ $dark: $slate;
 $light: $platinum;
 $info: $sea-green;
 
-$custom-colors: (
```

Comment:
This needs to be `$custom-colors` so we don't override the default `$theme-colors`. $custom-colors is merged into $theme-colors in `src/styles/bootstrap-theme/index.scss` in order to leave defaults in-tact.